03 Writing Algorithms/03 Securities/01 Key Concepts/07 Market Hours.html

```diff
@@ -4,11 +4,15 @@
 </p>
 
 <div class="section-example-container">
-    <pre class="csharp">var exchange = Securities[_symbol].Exchange;
+    <pre class="csharp"> // Check if the exchange is opened in the regular trading hours
```

Comment:
1. typos / "open"
2. Nonsense comment... the exchange will be open in regular trading hours. 

e.g. "Using exchange object, check if the exchange regular trading hours are open at the supplied time."